config/environments/production.rb

```diff
@@ -20,6 +20,12 @@
   # NGINX, varnish or squid.
   # config.action_dispatch.rack_cache = true
 
+  # Default from address for email
+  config.action_mailer.default_options = { from: ENV['RAILS_MAILER_DEFAULT_FROM_ADDRESS'] }
```

Comment:
@fonglh is there a place where we document all the environment variables which need to be set?